package/gluon-wifi-encrypt/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Gluon WiFi encrypt
+Encrypt wireless networks in a non-Freifunk setup.
+
+This package allows to encrypt 802.11s and infrastructure networks using a pre shared key (psk).
+
+This setup results in passwords being stored in plain-text. Site-configuration, Firmware-downloads and
+Devices must be protected against unauthorized access
+
+*make sure to use exclude hostapd-mini site.mk*
```

Comment:
#WLAN PSK encryption
Encrypts client and 802.11s WLAN networks using pre-shared keys.

This package allows to encrypt 802.11s and infrastructure networks using pre-shared keys (PSK).